<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogQSSCheckButton</class>
 <widget class="QDialog" name="DialogQSSCheckButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
	color: #606266;
	font-size: 14px;
}
QCheckBox:checked{
	color: #409eff;
}
QCheckBox:disabled{
	color: #c0c4cc;
}

QCheckBox::indicator{
	border-width: 1px;
	border-style: solid;
	border-radius: 2px;
}
QCheckBox::indicator:enabled{
	border-color: #dcdfe6;
}
QCheckBox::indicator:checked{
	border-color: #409eff;
	background-color: #409eff;
	image: url(:/images/7.png);
}
QCheckBox::indicator:disabled:checked{
	border-color: #dcdfe6;
	background-color: #f2f6fc;
	image: url(:/images/8.png);
}
QCheckBox::indicator:disabled:unchecked{
	border-color: #dcdfe6;
	background-color: #f2f6fc;
}
QCheckBox::indicator:indeterminate{
	border-color: #409eff;
	background-color: #409eff;
	image: url(:/images/9.png);
}</string>
   </property>
   <property name="title">
    <string>样式二</string>
   </property>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>已选中</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>未选中</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>部分选中</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>未选中禁用</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>选中禁用</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
	color: #606266;
	font-size: 14px;
}
QCheckBox:checked{
	color: #409eff;
}
QCheckBox:disabled{
	color: #c0c4cc;
}

QCheckBox::indicator{
	border-width: 1px;
	border-style: solid;
}
QCheckBox::indicator:enabled{
	border-color: #dcdfe6;
}
QCheckBox::indicator:checked{
	border-color: #409eff;
	border-width: 4px;
	background-color: #fff;
}
QCheckBox::indicator:disabled:checked{
	border-color: #dcdfe6;
	background-color: #f2f6fc;
}
QCheckBox::indicator:disabled:unchecked{
	border-color: #dcdfe6;
	background-color: #f2f6fc;
}
QCheckBox::indicator:indeterminate{
	border-color: #409eff;
	background-color: #fff;
	border-color: #409eff;
	border-top-width: 6px;
	border-bottom-width:6px;
	border-left-width:3px;
	border-right-width:3px;
}</string>
   </property>
   <property name="title">
    <string>样式一</string>
   </property>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>已选中</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>未选中</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>部分选中</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_9">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>未选中禁用</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_10">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>选中禁用</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
	color: transparent;
	background-color: #ff4949;
	border: none;
	border-radius: 10px;
}
QCheckBox:checked{
	background-color: #13ce66;
}
QCheckBox:disabled{
	background-color: #dcdfe6;
}
QCheckBox:checked:disabled{
	background-color: #8ae0b7;
}
QCheckBox:indeterminate{
	background-color:#ffc145
}

QCheckBox::indicator{
	width: 16px;
	height: 16px;
	border: none;
	border-radius: 8px;
	background-color: #fff;
	left: 2px;
}
QCheckBox::indicator:checked{
	left: 22px;
}
QCheckBox::indicator:indeterminate{
	left: 12px;
}
</string>
   </property>
   <property name="title">
    <string>样式三 开关</string>
   </property>
   <widget class="QCheckBox" name="checkBox_16">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>已选中</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>未选中</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_18">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>已选中</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_19">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>已选中</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_20">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>已选中</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>样式四 默认样式</string>
   </property>
   <widget class="QCheckBox" name="checkBox_11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>已选中</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>未选中</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_13">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>部分选中</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_14">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>未选中禁用</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_15">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>选中禁用</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
